core/vibes/soul/primitives/products-carousel/index.tsx

```diff
@@ -58,17 +66,32 @@ export function ProductsCarousel({
             <CarouselContent className="mb-10">
               {products.map((product) => (
                 <CarouselItem
-                  className="basis-full @md:basis-1/2 @lg:basis-1/3 @2xl:basis-1/4"
+                  className="basis-[calc(100%-1rem)] @md:basis-[calc(50%-0.75rem)] @lg:basis-[calc(33%-0.5rem)] @2xl:basis-[calc(25%-0.25rem)]"
                   key={product.id}
                 >
-                  <ProductCard product={product} />
+                  <ProductCard
+                    aspectRatio={aspectRatio}
+                    colorScheme={colorScheme}
+                    product={product}
+                  />
                 </CarouselItem>
               ))}
             </CarouselContent>
-            <div className="flex w-full items-center justify-between">
-              <CarouselScrollbar label={scrollbarLabel} />
-              <CarouselButtons nextLabel={nextLabel} previousLabel={previousLabel} />
-            </div>
+            {(showButtons || showScrollbar) && (
```

Comment:
If we're combining the values here why not just have 1 prop? Shouldn't we be using this separately for the scrollbar and the buttons?

Reply:
It's because I want to hide the entire row (that contains the margin) if both controls are hidden. Andrew wants to be able to hide them independently though.

Reply:
I missed we were using these in the classes below as well 👍🏻